polkadot/runtime/common/src/auctions/tests.rs

```diff
@@ -0,0 +1,1034 @@
+use super::*;
```

Comment:
exact same comments here in tests:

- copyright
- minimal description
- feature flag